### PR TITLE
Added one developer in tutorials

### PR DESCRIPTION
**Pull request by a-plus-coding**
Added my channel to developers list. I have made tutorials mainly using python and pygame but am planning to make more and will continue to update the channel.

### CODE REVIEW

**Comment by ErikCH** [comment]
Please put it at the end of the list and I'll merge @a-plus-coding :)

File: README.md [hunk 1]
```diff
@@ -19,6 +19,7 @@
 ### Tutorial
 > These channels focus on tutorials and code alongs.
 
+* [A Plus Coding](http://https://www.youtube.com/channel/UC-pLL_SFEm6B9Eeyr-V424Q)
 * [Program With Erik](http://erik.video)
```
Comment (scanf) [inline README.md]: This not a valid link, you have `http://https://`. For it to be a valid URL you need to pick one, either `http` or `https`, not both.

**Comment by sukhchainn** [comment]
What is LGTM?

**Comment by ErikCH** [comment]
@sukhchainn (Looks Good To Me)

**Comment by scanf** [comment]
@a-plus-coding in case you are not familiar with how to resolve merge conflicts see this guide from GitHub https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/resolving-a-merge-conflict-on-github

I noticed someone else was having issues resolving the conflicts so if that's not the case for you feel free to ignore this :)

**Comment by ErikCH** [comment]
I fixed it LGTM

**Pull request merged**